pkg/obfuscate/sql_test.go

```diff
@@ -2057,5 +2057,176 @@ func TestToUpper(t *testing.T) {
 			}
 		})
 	}
+}
```

Comment:
since there are two obfuscation/normalization flags plus an existing default obfuscation codepath, maybe add a test that shows behavior with combinations of options?